viewing_party/party.py

```diff
@@ -1,23 +1,119 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    inputs = [title, genre, rating]
+    for input in inputs:
+        if input == None:
+            return None
+    new_movie = {}
+    new_movie["title"] = title

```

Comment:
This totally works but you can also return a dictionary rather than create an empty one, assign it to variable, then populate it key/value by key/value, like so:
```python
return {
    "title": title,
    "genre": genre,
    "rating": rating
}
```